src/core/connection.ts

```diff
@@ -14,10 +14,10 @@ export class Connection extends EventEmitter {
   connected: boolean;
   redis: IORedis.Redis | IORedis.Cluster;
 
-  constructor(options: ConnectionOptions = {}) {
+  constructor(options: any = {}) {
```

Comment:
Why remove a stong type for an `any`?

Reply:
This one is not clever I agree. Few line under, I've try to type the "for", and impossible, because it's a for on a key... 

For clean it, we have to change the way of the variables affectation.